lib/galaxy/tool_util/deps/commands.py

```diff
@@ -118,16 +118,12 @@ def _wait(cmds, **popen_kwds):
     return stdout
 
 
-def download_command(url, to=STDOUT_INDICATOR, quote_url=False):
+def download_command(url, to=STDOUT_INDICATOR):
```

Comment:
quote_url is still used in Planemo right? Why break backward compatibility on this?

Reply:
Forgot about that! It's quite unsecure though; if I fix it in Planemo, would it be OK? Or would you like to keep it any way?

Reply:
Done in https://github.com/galaxyproject/planemo/pull/984